daemon/src/main.rs

```diff
@@ -68,4 +71,13 @@ async fn main() {
         }
         Err(_) => eprintln!("Could not connect to server. Possible it does not exist yet."),
     }
+
+    // tonic gRPC server for the communication with the client
+    let addr = "0.0.0.0:5768".parse()?;
+    let cd_svc = client_server::ClientDaemonService {};
+    let cd_srv = ClientDaemonServer::new(cd_svc);
+    println!("Client gRPC server on {}", addr);
```

Comment:
It looks like you implement the connection here, is it work?

Reply:
* working

Reply:
Yep! If you run it, you can use a gRPC client like (BloomRPC)[https://github.com/bloomrpc/bloomrpc] and import `proto/client_daemon.proto` and fire off RPC calls, though all of them are just blank for now.